express-get/server.ts

```diff
@@ -0,0 +1,64 @@
+import express from 'express';
+import pg from 'pg';
+import { ClientError, errorMiddleware } from './lib/index.js';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/pagila',
+  ssl: {
+    // Allow non-SSL traffic to localhost
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+
+app.get('/api/films', async (req, res, next) => {
+  try {
+    const sql = `
+    select "description",
+      "filmId",
+      "replacementCost",
+      "title"
+    from "films"
+    order by "replacementCost" desc
+    limit 2
+    `;
+    const results = await db.query(sql);
+    const films = results.rows;
+    if (!films) {
```

Comment:
When getting all films, you would just return an empty array instead of a 404. So this check isn't needed.